src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -79,18 +82,32 @@ public function doRun(InputInterface $input, OutputInterface $output): int
 
     protected function doRunCommand(Command $command, InputInterface $input, OutputInterface $output): int
     {
+        $requestStack = null;
+        $renderRegistrationErrors = true;
```

Comment:
can be removed now

Reply:
Why?

Reply:
why add it in the first place?

Reply:
Because in the old code there is an early return in the first if, and a duplicated if where I used the variable below.